### PR TITLE
raspimouse_ros2_examples: 2.0.0-1 in 'humble/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by YusukeKato**
Increasing version of package(s) in repository `raspimouse_ros2_examples` to `2.0.0-1`:

- upstream repository: https://github.com/rt-net/raspimouse_ros2_examples.git
- release repository: https://github.com/ros2-gbp/raspimouse_ros2_examples-release.git
- distro file: `humble/distribution.yaml`
- bloom version: `0.11.2`
- previous version for package: `null`

## raspimouse_ros2_examples

```
* Humble対応 (#41 <https://github.com/rt-net/raspimouse_ros2_examples/issues/41>)
* Contributors: Shuhei Kozasa
```

### CODE REVIEW

**Review by clalancette — APPROVED** [review]
This looks reasonable to me, but @audrow I'd like a second opinion on the package name before we merge.

**Review by audrow — APPROVED** [review]

**Pull request merged**
